### PR TITLE
feat: add new space settings for nft claimer

### LINKED ISSUE #822: Add new space setting for NFT Claimer
New settings for space are needed for the new NFT Claimer project, PR coming

### PR DESCRIPTION
**Pull request by wa0x6e**
Fixes #822

Changes proposed in this pull request:
Add new space settings for NFT claimer

New settings are nested in a `nftClaimer` key. Feel free to propose a new key name if it's not suitable

### CODE REVIEW

File: src/schemas/space.json [hunk 1]
```diff
@@ -326,6 +326,39 @@
...
+              "maxLength": 42
+            },
+            "network": {
+              "type": "string",
+              "title": "Network",
+              "maxLength": 12
+            },
+            "enabled": {
+              "type": "boolean"
+            }
```
Comment (ChaituVR) [inline src/schemas/space.json:359]: I am not following much but do we need `enabled`? if we have `nftClaimer` other than object, can we consider it as enabled? or do we want to keep the settings when it is disabled?

Comment (wa0x6e) [reply src/schemas/space.json:359]: NFT Claimer can be enabled, then disabled later

Comment (ChaituVR) [reply src/schemas/space.json:359]: I mean if it is disabled, do we want to keep the previous settings?

Comment (wa0x6e) [reply src/schemas/space.json:359]: yes, if because he can enable it back

Comment (wa0x6e) [reply src/schemas/space.json:359]: And there may be already minted nft

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/schemas/space.json (hunk 1 above)
Comment (samuveth) [reply src/schemas/space.json:359]: I'd suggest you add Title, Description and Examples to each field, then you will be able to use `TuneForm` to easily generate the form on the Ui. Example here: https://tune-histoire.vercel.app/ see `Source` in the sidebar for the used Schema

Comment (wa0x6e) [reply src/schemas/space.json]: Added

File: src/schemas/space.json [hunk 2]
```diff
@@ -330,20 +330,31 @@
           "type": "object",
           "properties": {
             "maxSupply": {
-              "type": "number",
-              "minimum": 0
+              "type": "integer",
```
Comment (samuveth) [inline src/schemas/space.json:333]: Why integer and not number?

Comment (wa0x6e) [reply src/schemas/space.json:333]: Because it's an integer, and not floating number

Comment (samuveth) [reply src/schemas/space.json:333]: I see

**Review by samuveth — APPROVED** [review]
utACK

**Comment by wa0x6e** [comment]
Obsolete PR
All settings have been migrated to be store on-chain, and retrieve via subgraph

**Pull request closed without merging**